src/diagnostics/translation.md

```diff
@@ -87,9 +87,9 @@ information that needs to be provided by the code to do so.
 
 ### Compile-time validation and typed identifiers
 rustc's Fluent resources for the default locale (`en-US`) are in the
-[`compiler/rustc_error_messages/locales/en-US`] directory. Currently, each crate
-which defines translatable diagnostics has its own Fluent resource, such as
-`parser.ftl` or `typeck.ftl`.
+[`compiler/rustc_error_messages/messages.ftl`] file. Currently, each crate
+which defines translatable diagnostics has its own Fluent resource in
+a file named `messages.ftl`.
```

Comment:
I'm not actually up to speed on the Fluent stuff, so this may not be worded the best. It was converted from a directory to a file in https://github.com/rust-lang/rust/pull/103042, and then renamed again in https://github.com/rust-lang/rust/pull/108682.

Reply:
This would be correct if the first sentence was removed. I would also add an example or two:
> ... such as `compiler/rustc_borrowck/messages.ftl` and `compiler/rustc_parse/messages.ftl`

Reply:
Sure, I went ahead and pushed a change. Skimming over the chapter, it looks like there are quite a few things that are out-of-date. I don't know if there is someone involved in translation who could be pinged to update it.

Reply:
I noticed, and intend to update it soon.